code/modules/cargo/exports/large_objects.dm

```diff
@@ -125,19 +126,46 @@
 								/datum/gas/antinoblium,
 								/datum/gas/halon
 								)
+	if(gases[/datum/gas/hypernoblium][MOLES] > 0)
+		worth += round((5/k_elasticity) * (1 - NUM_E**(-1 * k_elasticity * gases[/datum/gas/hypernoblium][MOLES])))
```

Comment:
Can we de-duplicate all this code please? And use a single list to key in the `x / k_elasticity` factors.

Reply:
should be done